app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt

```diff
@@ -1605,6 +1631,30 @@ class AddProfileActivityTest {
     return ActivityResult(RESULT_OK, resultIntent)
   }
 
+  /**
+   * Appends the specified text to a view. This is needed because Robolectric doesn't seem to
+   * properly input digits for text views using 'android:digits'. See
+   * https://github.com/robolectric/robolectric/issues/5110 for specifics.
+   */
+  private fun appendText(text: String): ViewAction {
```

Comment:
Suggesting a TODO for this
https://github.com/oppia/oppia-android/issues/1840

Reply:
Done